<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">YAIPT!</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/yaipt.png</normaloff>images/yaipt.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout1">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolbar">
   <property name="windowTitle">
    <string notr="true"/>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPlane"/>
   <addaction name="actionEqualize"/>
   <addaction name="actionDenoise"/>
   <addaction name="actionInterpolate"/>
   <addaction name="actionDFT"/>
   <addaction name="actionButterworth"/>
   <addaction name="actionCanny"/>
   <addaction name="actionMorphology"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string notr="true">Open</string>
   </property>
  </action>
  <action name="actionDFT">
   <property name="text">
    <string notr="true">DFT</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string notr="true">Help</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string notr="true">Save</string>
   </property>
  </action>
  <action name="actionPlane">
   <property name="text">
    <string notr="true">Plane</string>
   </property>
  </action>
  <action name="actionEqualize">
   <property name="text">
    <string notr="true">Equalize</string>
   </property>
  </action>
  <action name="actionDenoise">
   <property name="text">
    <string notr="true">Denoise</string>
   </property>
  </action>
  <action name="actionInterpolate">
   <property name="text">
    <string notr="true">Interpolate</string>
   </property>
  </action>
  <action name="actionButterworth">
   <property name="text">
    <string notr="true">Butterworth</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string notr="true">Canny</string>
   </property>
  </action>
  <action name="actionMorphology">
   <property name="text">
    <string notr="true">Morphology</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
